Backend/env/variable.js

```diff
@@ -0,0 +1 @@
+module.exports = "basic bTpw";
```

Comment:
make sure filenames are clear and represent what they contain, see style guide for guidance on naming